config/webpack/webpack.config.js

```diff
@@ -65,6 +65,38 @@ const makeCssLoaders = (options = {}) => {
   ];
 };
 
+const makeCssInJsLoaders = (options = {}) => {
+  const { server = false } = options;
+
+  return [
+    {
+      loader: require.resolve(`css-loader${server ? '/locals' : ''}`),
+      options: {
+        modules: true,
+        importLoaders: 3
+      }
+    },
+    {
+      loader: require.resolve('postcss-loader'),
```

Comment:
Sorry, looking at this again, it looks like we also need shared `cssLoaders` too?